src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/AdditionalConditionRequest.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model
+
+import io.swagger.v3.oas.annotations.media.Schema
+
+@Schema(description = "Describes an additional condition")
+data class AdditionalConditionRequest(
+  @Schema(description = "Coded value for the additional condition", example = "meetingAddress")
+  val code: String? = null,
+
+  @Schema(description = "The category of the additional condition", example = "Freedom of movement")
+  val category: String? = null,
+
+  @Schema(description = "Sequence of this additional condition within the additional conditions", example = "1")
+  val sequence: Int? = null,
+
+  @Schema(description = "The textual value for this additional condition", example = "You must not enter the location [DESCRIPTION]")
+  val text: String? = null,
+
```

Comment:
I think category and text could be derived from the code alone

I think index could possibly be derived from position in array as this overwrites all existing conditions